contact-us.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Us</title>
+</head>
+
+<body>
+    <section>
+        <div class="container">
+            <div class="contactinfo">
+                <div>
+                    <h2>Contact Info</h2>
+                    <ul class="info">
+                        <li>
+                            <span><img src="https://cdn.picpng.com/pin/pin-location-map-icon-53136.png" alt=""></span>
+                            <span>Maharashtra, Navi Mumbai <br>
+                                Kharghar <br>
+                                sector 35C <br>
+                                pincode:410210 </span>
+                        </li>
+
+                        <li>
+                            <span><img
+                                    src="https://i2.wp.com/ctcnl.com/wp-content/uploads/2017/04/mail-icon-rood-RGB.png?ssl=1"
+                                    alt=""></span>
+                            <span>saadmahimkar@icloud.com</span>
+
+                        </li>
+
+                        <li>
+                            <span><img src="https://www.seekpng.com/png/full/136-1360951_call-png.png" alt=""></span>
+                            <span>8888-**-44-**</span>
+
+                        </li>
+                    </ul>
+                </div>
+                    <ul class="soc">
+
+
+                        <li><a href="#"><img
+                                src="https://th.bing.com/th/id/OIP.1PhXG_4NLoBd-AdVO5DD7QHaHa?pid=ImgDet&rs=1"></a></li>
+                        <li><a href="#"><img
+                                src="https://www.verfvanniveau.nl/wp-content/uploads/2019/08/logo-social-fb-facebook-icon.png"></a>
+                        </li>
+                        <li><a href="#"><img
+                                src="https://th.bing.com/th/id/R.21fba12531ac377e1642a01921db9fbd?rik=ErbH90Kfy3L6tg&riu=http%3a%2f%2fencomium.ng%2fwp-content%2fuploads%2f2016%2f01%2fLinkedIn-Logo.png&ehk=qo9l10kLERwTTr5WEWyHeVR%2bKIWh4CchVBnnAxyz3ic%3d&risl=&pid=ImgRaw&r=0"></a>
+                        </li>
+                        <li><a href="#"><img src="https://jesselumarie.com/blog/theme/images/insta_logo.png"></a>
+                        </li>
+
+                    </ul>
+
+
```

Comment:
In html we don't need much spacing. Indentation helps us reading the code easily.